packages/suite/src/components/wallet/CoinmarketLayout/components/AccountTransactions/components/SpendTransaction/index.tsx

```diff
@@ -0,0 +1,115 @@
+import { TradeSpend } from '@suite/reducers/wallet/coinmarketReducer';
```

Comment:
```suggestion
import { TradeSpend } from '@wallet-reducers/coinmarketReducer';
```
and pls move it below

Reply:
Damn, forgot to move this one below. Second fix-up on its way.